scripts/vpn_kill_users.py

```diff
@@ -6,22 +6,33 @@
 #
 # Contributors:
 # Guillaume Destuynder <gdestuynder@mozilla.com>
+# Greg Cox <gcox@mozilla.com>
 #
-# Note: parsing and matching are a little dirty, but so is the OpenVPN management protocol. So yeah :)
-# This works as of OpenVPN 2.3.4. It should work in later versions too, hopefully - unless/until syntax changes.
+# Note: parsing and matching are a little dirty,
+#       but so is the OpenVPN management protocol.
+# This works as of OpenVPN 2.4.6.
 #
 # Recommended openvpn server settings:
 # management /var/run/openvpn-udp-stage.socket unix
 # management-client-group vpnmgmt
 
-import socket
-import mozlibldap
+import socket as socket_module
```

Comment:
It stems from a cleanup of ago with way too many uses of the word socket in multiple contexts.  Noted for a next-rev, since this will get another pass-through when we get rid of RHEL6 and py2.6.